and-cli-github.js

```diff
@@ -16,23 +20,108 @@ require("./command-runner").run(async () => {
     program
         .usage("option")
         .description(github.description())
+        .option(
+            "--add-topic <topic>",
+            "Add topic to specified repository, or all AndcultureCode repositories if no repo provided"
+        )
+        .option(
+            "--list-topics",
+            "List topics for a given repo (used in conjunction with --repo)"
+        )
+        .option("--list-repos", "Lists all andculture repos")
```

Comment:
For consistency consider changing "andculture" to "AndcultureCode".

Reply:
I'm also a culprit of switching between 'andculture' and 'andculturecode'. Perhaps we make a constant for CLI and interpolate it throughout the application to settle it?